CMakeLists.txt

```
@@ -1,27 +1,122 @@
-cmake_minimum_required(VERSION 3.0)
-
+cmake_minimum_required(VERSION 3.15)
 project(livox_sdk)
 
+# C++ config.
 set(CMAKE_CXX_STANDARD 11)
```

Since we're using C++17 in other packages too:

```suggestion
set(CMAKE_CXX_STANDARD 17)
```